<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartAid</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

/*-----QWidget-----*/
QWidget
{
	background-color:#fff;
	color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #c2c7d5;
	font-size: 13px;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));
	color: #fff;
	font-size: 11px;
	font-weight: bold;
	border: none;
	border-radius: 3px;
	padding: 5px;

}


QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 5px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;
    border: 1px solid #607cff;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #08b099;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #c2c7d5;
	color: #000;
	font-weight: bold;
	border: none;
	border-radius: 2px;
	padding: 3px;

}


/*-----QListView-----*/
QListView
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(50, 61, 80, 255),stop:1 rgba(44, 49, 69, 255));
	color: #fff;
	font-size: 18px;
	font-weight: bold;
	border: 1px solid #191919;
	show-decoration-selected: 0;
	outline : 0;

}


/*QListView::item
{
	color: #31cecb;
	background-color: #454e5e;
	border: none;
	padding: 5px;
	border-radius: 15px;
	padding-left : 10px;
	height: 52px;

}*/

QListView::item
{
	color: #31cecb;
	background-color: #bf7069;
	border: none;
	padding: 5px;
	border-radius: 15px;
	padding-left : 10px;
	height: 62px;

}




/*QListView::item:selected
{
	color: #31cecb;
	background-color: #454e5e;
}*/


QListView::item:selected
{
	color: #ffabab;
	background-color: #bf7069;
}

QListView::item:!selected
{
	color: #2C3E50;
	/*color:white;*/
	background-color: transparent;
	border: none;
	padding-left : 10px;

}



QListView::item:!selected:hover
{
	color: #bbbcba;
	background-color: #454e5e;
	border: none;
	padding-left : 10px;

}


/*-----QTreeView-----*/
QTreeView 
{
	background-color: #232939;
	show-decoration-selected: 0;
	color: #c2c8d7;

}


QTreeView::item 
{
	border-top-color: transparent;
	border-bottom-color: transparent;

}


QTreeView::item:hover 
{
	background-color: #606060;
	color: #fff;

}


QTreeView::item:selected 
{
	background-color: #0ab19a;
	color: #fff;

}


QTreeView::item:selected:active
{
	background-color: #0ab19a;
	color: #fff;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #232939;
	border: 1px solid gray;
    color: #f0f0f0;
    gridline-color: #232939;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #606060;
    color: #f0f0f0;

}


QTableView::item:selected 
{
	background-color: #0ab19a;
    color: #F0F0F0;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
	background-color: #343a49;
    color: #fff;

}


QHeaderView::section
{
	color: #fff;
	border-top: 0px;
	border-bottom: 1px solid gray;
	border-right: 1px solid gray;
	background-color: #343a49;
    margin-top:1px;
	margin-bottom:1px;
	padding: 5px;

}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #0ab19a;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: transparent;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: transparent;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:transparent;
/*background-color: #262626;*/
border-radius:15px;
font-family:century gothic;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>971</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressources.qrc">:/icons/icons/icone.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*color: #ff8b8b;*/
color:#d46;
font-size:30px;
font:bold;
font-family : century gothic;</string>
      </property>
      <property name="text">
       <string>SmartAid Association</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>891</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:50px;

background-color: rgb(255, 228, 228);

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="user_pic">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>451</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#fff;
border-radius:50px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>30</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2C3E50;
font-size:30px;
font:bold;
font-family : century gothic;</string>
       </property>
       <property name="text">
        <string>John Doe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>70</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #4C5958;
font-size:18px;
font-family : century gothic;</string>
       </property>
       <property name="text">
        <string>john.doe@esprit.tn</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>451</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>page2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_6"/>
    <widget class="QWidget" name="page_7"/>
    <widget class="QWidget" name="page_8">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>841</width>
        <height>461</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string> Ajouter Crise</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>84</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>184</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="lineEdit_description">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_datec">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumDate">
         <date>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>330</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Niveau</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>400</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>380</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>260</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="lineEdit_Adresse">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>230</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_niveau">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>330</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>380</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Traiter</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>En cours</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>380</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Non traiter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Afficher">
       <attribute name="title">
        <string>Afficher Crises</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>682</x>
          <y>400</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Charger les données</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>811</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Modifier Crise</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>280</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>400</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Traiter</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_niveau_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>350</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>104</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_mod_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>400</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>En cours</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_datec_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumDate">
         <date>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_mod_3">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>400</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Non traiter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>204</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="lineEdit_Adresse_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="lineEdit_description_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>171</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>350</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Niveau</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modif">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_mod">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Supprimer Crise</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supp">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
